src/server/middleware.js

```diff
@@ -25,6 +25,7 @@ export default function (configDir) {
     noInfo: true,
     publicPath: config.output.publicPath,
     watchOptions: config.watchOptions || {},
+    ...config.devServer,
```

Comment:
Where can I pass the `config.devServer` to, to manipulate these configurations?

Reply:
Should be in the webpack.config.js export I'm pretty sure.